seed/data_importer/tasks.py

```diff
@@ -779,6 +779,12 @@ def _save_greenbutton_data_task(readings, meter_id, meter_usage_point_id, progre
     meter = Meter.objects.get(pk=meter_id)
 
     result = {}
+    result_summary_key = "{} - {} - {}".format(
```

Comment:
Builds the summary key once in the beginning vs twice in the two most likely cases below.